In [2]:
library(tidyverse)
library(repr)

players <- read_csv("data/players.csv")
sessions <- read_csv("data/sessions.csv")


Rows: 196 Columns: 7
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (4): experience, hashedEmail, name, gender
dbl (2): played_hours, Age
lgl (1): subscribe

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
Rows: 1535 Columns: 5
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (3): hashedEmail, start_time, end_time
dbl (2): original_start_time, original_end_time

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.


PLAYERS

The dataset consists of characteristics of players in the Minecraft server. It consists of 198 observations and 7 variables:
- experience (chr)
    - The player's experience level, consisting of: Amateur, Beginner, Regular, Veteran, and Pro.
- subscribe (lgl)
    - Whether or not the player is subscribed to the game newsletter, can be TRUE or FALSE.
- hashedEmail (chr)
    - The player's hashed email.
- played_hours (dbl)
    - The number of hours the player has played in this Minecraft server.
    - A potential issue with the way this data was collected is that it only represents the number of hours the player has played in this specific server, and is not representative of their total number of hours played, which can lead to an inaccurate representation of their commitment.
- name (chr)
    - The player's first name.
- gender (chr)
    - The player's gender, can be Male, Female, Non-binary, Two-spirited, Agender, Prefer not to say, or Other.
    - The "Prefer not to say" values pose a potential issue when deciding whether or to treat them as NA values or as their own category.
- Age (dbl)
    - The player's age in years.
    - This column contains some NA values, which will need to be removed when performing data manipulation and analysis.



SESSIONS

The dataset consists of data on individual play session by each player on the Minecraft server. It consists of 1535
observations and 5 variables:
- hashedEmail (chr)
    - The player's hashed email.
- start_time (chr)
    - The start time of the session.
- end_time (chr)
    - The end time of the session.
- original_start_time (dbl)
    - The original start time of the session.
- original_end_time (dbl)
    - The original end time of the session.

The start_time and end_time columns pose an issue because there are too many variables within 1 column (day, month, year, and time), so wrangling will need to be performed to make the data tidy.
